README.rst

```diff
@@ -97,7 +101,7 @@ To build Lightning plugins from source you can run
     PL_BACKEND=${PL_BACKEND} pip install pybind11 pennylane-lightning --no-binary :all:
 
 where ``${PL_BACKEND}`` can be ``lightning_qubit`` (default), ``lightning_gpu``,  ``lightning_kokkos``, or ``lightning_tensor``.
-The `pybind11 <https://pybind11.readthedocs.io/en/stable/>`_ library is required to bind the C++ functionality to Python.
+The `pybind11 <https://pybind11.readthedocs.io/en/stable/>`_ library is required to bind the C++ functionality to Python. If installing Lightning-GPU, Lightning-Tensor, or Lightning-Kokkos, additional dependencies may be required. We recommend referring to the respective guides for `Lightning-GPU installation <https://docs.pennylane.ai/projects/lightning/en/stable/lightning_gpu/installation.html>`_, `Lightning-Tensor installation <https://docs.pennylane.ai/projects/lightning/en/stable/lightning_tensor/installation.html>`_, and `Lightning-Kokkos installation <https://docs.pennylane.ai/projects/lightning/en/stable/lightning_kokkos/installation.html>`_.
```

Comment:
Reminder to use `:doc:` for internal linking here, rather than static links. Using `:doc:` will automatically keep readers on stable/latest

Reply:
This goes for the internal links below as well :)

Reply:
@josh146 I was trying to fix this one too! But this is the library's README file and the installation pages are not properly rendered when checking README from the repository main page. What's your suggestion?

Reply:
ahhh, good point @maliasadi. Because of the readme being rendered on GitHub, we have no choice here but to use a static link. That's a shame.

For other rst pages though that only appear in the documentation, we should use :doc: